syfoalerts.yaml

```diff
@@ -24,6 +24,13 @@ spec:
       action: "Se `kubectl describe pod {{ $labels.kubernetes_pod_name }}` for events, og `kubectl logs {{ $labels.kubernetes_pod_name }}` for logger"
       sla: respond within 1h, during office hours
       severity: danger
+    - alert: syfo-kontinuerlig-restart
+      expr: sum(increase(kube_pod_container_status_restarts_total{container=~"syfoarbeidsgivertilgang|syfonarmesteleder|syfoservicestrangler|syfosoknad|syfosyketilfelle|syfosmregler|syfosminfotrygd|syfosmregister|pdf-gen|syfogsak|syfoinntektsmelding|syfosoknadvarsel|syfoaltinn|syfovarsel|syfosmoppgave|syfosmarena|syfosmvarsel|syfosmsak|syfosmmottak"}[1h])) > 3
+      for: 5m
+      description: "Flere pods restarter i {{ $labels.kubernetes_namespace }} siste timen. {{ $labels.app }} var den siste som gjorde det"
```

Comment:
Er labelen for 'app' tilgjengelig i denne spørringen? I 'syfo-mangler-metrikker' er ikke app-labelen tilgjengelig iallefall. Når labelen ikke er tilgjengelig ser du ikke hvilken app som trigger alert, du ser bare alerten. For 'syfo-mangler-metrikker' ble vi enige om at detvar greit nok siden den trigges svært sjelden, men for restart tror jeg det er nyttig å vite hvilken app det gjelder. Vet ikke om det er noen annen måte å få vist app på enn å splitte alerten i en pr app.

Reply:
Nei det har du rett i. Fikset i 083795e